jest.config.js

```diff
@@ -5,9 +5,13 @@ const { compilerOptions } = require('./tsconfig');
 module.exports = {
   preset: 'react-native',
   setupFiles: ['./config/test/jest-setup.js'],
+  testPathIgnorePatterns: ['node_modules', 'e2e'],
   transform: {
     '\\.tsx?$': 'ts-jest',
   },
+  transformIgnorePatterns: [
+    'node_modules/(?!((jest-)?react-native|react-native-keyboard-area|@react-native(-community)?)/)',
+  ],
```

Comment:
Anything matched here gets transpiled. There are a few react-native libs that ship untranspiled code.